events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+
+            currentChoices = [];
+            currentChoices.push("True");
+            currentChoices.push("False");
+            lights.forEach(function(light) {
+                var lightName = Entities.getEntityProperties(light, 'name').name;
+                if (lightName.indexOf("Green") !== -1) {
+                    Entities.callEntityServerMethod(light, "lightsOff");
+                } else if (lightName.indexOf("Blue") !== -1) {
+                    Entities.callEntityServerMethod(light, "lightsOff");
+                }
+            });
+        } else {
+            currentChoices = [];
+            currentChoices.push(triviaData[0].correct_answer);
+            triviaData[0].incorrect_answers.forEach(function(choice) {
+                currentChoices.push(choice);
+            });
+            shuffle(currentChoices);
+            currentChoices.forEach(function(choice, index) {
+                Entities.callEntityServerMethod(choiceTexts[index], "textUpdate", [choice, true]);
+            });
+        }        
+    }
+
+    function clearGame() {
+        correctCount = null;
+        prizeMoney = 0;
+        winnerID = null;
+
+        bubbleOff();
+        lights.forEach(function(light) {
+            Entities.callEntityServerMethod(light, "lightsOff");
+        });
+        correctHighlights.forEach(function(highlight) {
+            Entities.callEntityServerMethod(highlight, "lightsOff");
+        });
+        Entities.callEntityServerMethod(questionText, "textUpdate", ["Questions will appear here", true]);
+        choiceTexts.forEach(function(choice) {
+            Entities.callEntityServerMethod(choice, "textUpdate", ["Answers appear here", true]);
+        });
+        Entities.callEntityServerMethod(answerText, "textUpdate", ["", true]);
+        Entities.callEntityServerMethod(avatarCounter, "textUpdate", [0, true]);
+        Entities.callEntityServerMethod(prizeDisplay, "textUpdate", [0, true]);
+    }
+
+    function isPositionInsideBox(position, gameZoneProperties) {
+        var localPosition = Vec3.multiplyQbyV(Quat.inverse(gameZoneProperties.rotation),
+            Vec3.subtract(position, gameZoneProperties.position));
+        var halfDimensions = Vec3.multiply(gameZoneProperties.dimensions, HALF_MULTIPLIER);
+        return -halfDimensions.x <= localPosition.x &&
+                halfDimensions.x >= localPosition.x &&
+               -halfDimensions.y <= localPosition.y &&
+                halfDimensions.y >= localPosition.y &&
+               -halfDimensions.z <= localPosition.z &&
+                halfDimensions.z >= localPosition.z;
+    }
+
+    function usersInZone(gameZoneProperties) {
+        var count = 0;
+        AvatarManager.getAvatarIdentifiers().forEach(function(avatarID) {
+            var avatar = AvatarManager.getAvatar(avatarID);
+            if (avatar.sessionUUID) {
+                if (isPositionInsideBox(avatar.position, gameZoneProperties)) {
```

Comment:
use logical AND on line above this one instead of nesting conditionals

Reply:
fixed